packages/ketcher-react/src/script/ui/views/toolbars/LeftToolbar/Bond/Bond.tsx

```diff
@@ -28,31 +27,33 @@ import {
   groupDescriptors
 } from './options'
 
-interface BondProps extends Omit<ToolbarGroupItemProps, 'id' | 'options'> {}
+interface BondProps extends Omit<ToolbarGroupItemProps, 'id' | 'options'> {
+  height?: number
+}
 interface BondCallProps extends ToolbarGroupItemCallProps {}
 
 type Props = BondProps & BondCallProps
 
 const Bond = (props: Props) => {
-  const collapseBond = useMediaQuery({ query: '(max-height: 700px)' })
+  const { height, ...rest } = props
 
-  if (collapseBond) {
+  if (height && height <= 620) {
```

Comment:
Please move magic numbers to constants

Reply:
fixed in [move media sizes to constants](https://github.com/epam/ketcher/pull/323/commits/d9abe72d289ea5e2293030fbaa0060c266759e0c)